Wikipedia/Third Party Code/RMessage/RMessageView.m

```diff
@@ -451,13 +462,7 @@ - (void)setupDesign
 
 - (void)setupLayout
 {
-  self.translatesAutoresizingMaskIntoConstraints = NO;
-
-  // Add RMessage to superview
-  [self addToSuperviewForPresentation];
-
   if (self.messagePosition != RMessagePositionBottom) {
-    [self layoutIfNeeded];
```

Comment:
This is an unrelated cleanup thing - I think it was only needed with the way the old constraints were set up, which were modified in #4384. I don't see a need for this call at this point, plus we're calling `layoutIfNeeded` it again further down in the method after the constraints are set up. Removing it seems to cause no regressions in my testing.